src/rgw/rgw_iam_policy.cc

```diff
@@ -1479,6 +1482,9 @@ const char* action_bit_string(uint64_t action) {
   case s3BypassGovernanceRetention:
     return "s3:BypassGovernanceRetention";
 
+  case s3GetObjectAttributes:
+    return "s3:GetObjectAttributes";
+
```

Comment:
`action_bit_string()` is missing the corresponding case for s3GetObjectVersionAttributes

Reply:
ack